scripts/zkai_hybrid_proof_pressure_selector_gate.py

```
@@ -0,0 +1,695 @@
+    expect_equal(payload.get("decision"), DECISION, "decision")
+    expect_equal(payload.get("result"), RESULT, "result")
+    expect_equal(payload.get("issue"), ISSUE_URL, "issue")
+    if set(payload.get("non_claims", [])) != set(NON_CLAIMS):
+        raise HybridSelectorError("global non-claim drift")
+    rows = payload.get("selector_rows")
+    if not isinstance(rows, list) or not rows:
+        raise HybridSelectorError("selector rows missing")
+    seen: set[str] = set()
+    for row in rows:
+        if not isinstance(row, dict):
+            raise HybridSelectorError("selector row must be an object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)